### PR TITLE
Add cl_pocl_content_size extension

### PR DESCRIPTION
**Pull request by franz**
This is the OpenCL-Registry PR for the [OpenCL-Docs PR #663](https://github.com/KhronosGroup/OpenCL-Docs/pull/663)

### CODE REVIEW

File: extensions/registry.py [hunk 1]
```diff
@@ -351,4 +351,9 @@
         'flags' : { 'public' },
         'url' : 'extensions/amd/cl_amd_planar_yuv.txt',
     },
+    'cl_pocl_content_size' : {
+        'number' : 71,
+        'flags' : { 'public' },
+        'url' : 'extensions/pocl/cl_pocl_content_size.txt',
```
Comment (bashbaug) [inline extensions/registry.py:357]:
```suggestion
        'url' : 'extensions/pocl/cl_pocl_content_size.html',
```

I think this should have a html file extension instead?

You'll need to rebuild clext.php after making this change.

**Review by bashbaug — CHANGES_REQUESTED** [review]
I missed this PR, my apologies.

Mostly looks good but I think the filename needs a .html extension, not a .txt extension, or it will not link properly.

**Comment by franz** [comment]
Thanks Ben, i originally had the txt file, then i followed the current process to get the html, and somehow i missed the extension. I've updated `clext.php` so it should be all fixed now.

**Review by bashbaug — APPROVED** [review]
Thanks, LGTM!  Merging.

**Pull request merged**